L1Trigger/L1TMuonEndCap/plugins/L1TMuonEndCapTrackProducer.cc

```diff
@@ -65,13 +65,12 @@ void L1TMuonEndCapTrackProducer::produce(edm::Event& ev,
   //fprintf (write,"12345\n"); //<-- part of printing text file to send verilog code, not needed if George's package is included
 
   //std::auto_ptr<L1TMuon::InternalTrackCollection> FoundTracks (new L1TMuon::InternalTrackCollection);
-  auto FoundTracks = std::make_unique<L1TMuon::InternalTrackCollection>();
-  auto OutputCands = std::make_unique<l1t::RegionalMuonCandBxCollection>();
-  auto OutTracks = std::make_unique<l1t::EMTFTrackCollection>();
-  auto OutHits = std::make_unique<l1t::EMTFHitCollection>();
-  auto OutputTracks = std::make_unique<l1t::EMTFTrackExtraCollection>();
-  auto OutputHits = std::make_unique<l1t::EMTFHitExtraCollection>();
-  auto OutputHitsRPC = std::make_unique<l1t::EMTFHitExtraCollection>();
+  std::auto_ptr<l1t::RegionalMuonCandBxCollection > OutputCands (new l1t::RegionalMuonCandBxCollection);
+  std::auto_ptr<l1t::EMTFTrackCollection> OutTracks (new l1t::EMTFTrackCollection);
+  std::auto_ptr<l1t::EMTFHitCollection> OutHits (new l1t::EMTFHitCollection);
+  std::auto_ptr<l1t::EMTFTrackExtraCollection> OutputTracks (new l1t::EMTFTrackExtraCollection);
+  std::auto_ptr<l1t::EMTFHitExtraCollection> OutputHits (new l1t::EMTFHitExtraCollection);
+  std::auto_ptr<l1t::EMTFHitExtraCollection> OutputHitsRPC (new l1t::EMTFHitExtraCollection);
 
```

Comment:
My guess is that you made this PR to 8_1_X from your 8_0_X development without taking into account the differences. The commit of Vladimir exists in 8_1_X but not in 8_0_X.